roles/inspec/tasks/main.yml

```diff
@@ -61,13 +62,12 @@
     dest: /etc/inspec/host-controls/inspec.yml
     mode: 0644
 
-- name: inspec controller control files
+- name: inspec openstack control files
   template:
-    src: '{{ item }}'
-    dest: /etc/inspec/host-controls/controls/controller/
+    src: etc/inspec/host-controls/controls/openstack-security.rb
+    dest: /etc/inspec/host-controls/controls/
     mode: 0644
-  with_fileglob: ../templates/etc/inspec/host-controls/controls/controller/*
-  when: inventory_hostname in groups['controller']
+  when: inventory_hostname in groups['controller'] or  inventory_hostname in groups['compute'] or inventory_hostname in groups['cinder_volume']
```

Comment:
This logic can be simplified to 
`inventory_hostname in groups['controller','compute','cinder_volume']`